Source/WebCore/animation/ViewTimeline.h

```diff
@@ -47,6 +47,12 @@ struct ViewTimelineInsets {
 };
 
 class ViewTimeline final : public ScrollTimeline {
+    struct ViewTimelineData {
+        float scrollContainerSize;
```

Comment:
Initialize these to zero.

Reply:
Could this struct be private? You could also rename it to just `Data` since it's within the `ViewTimeline` scope.

Reply:
Fixed.